test/TesterAzureUtils/Persistence/PersistenceGrainTests_AzureTableStore.cs

```diff
@@ -41,7 +41,6 @@ protected override TestCluster CreateTestCluster()
                 Guid serviceId = Guid.NewGuid();
                 var options = new TestClusterOptions(initialSilosCount: 4);
                 options.ClusterConfiguration.Globals.DataConnectionString = TestDefaultConfiguration.DataConnectionString;
-                options.ClusterConfiguration.Globals.LivenessType = GlobalConfiguration.LivenessProviderType.AzureTable;
```

Comment:
I see that you dropped this, but I don't see that you are using the SiloBuilder extension method either. Is this being set somewhere else?
I see the same thing in `PersistenceGrainTests_AzureBlobStore`